Description: Add hppa support
Author: John David Anglin <dave.anglin@bell.net>
Origin: http://bugs.debian.org/730398

--- a/mingw-w64-tools/widl/include/basetsd.h
+++ b/mingw-w64-tools/widl/include/basetsd.h
@@ -329,6 +329,8 @@
 # undef  WORDS_BIGENDIAN
 #elif defined(__s390__) || defined(__s390x__)
 # define WORDS_BIGENDIAN
+#elif defined(__hppa__)
+# undef  WORDS_BIGENDIAN
 #elif !defined(RC_INVOKED) && !defined(__WIDL__) && !defined(__midl)
 # error Unknown CPU architecture!
 #endif
